ray-operator/Makefile

```diff
@@ -114,7 +114,7 @@ controller-gen: ## Download controller-gen locally if necessary.
 
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
-	test -s $(KUSTOMIZE) ||  GOBIN=$(KUSTOMIZE)/.. go install sigs.k8s.io/kustomize/kustomize/v3@v3.10.0
+	test -s $(KUSTOMIZE) ||  GOBIN=$(KUSTOMIZE)/.. go install sigs.k8s.io/kustomize/kustomize/v5@v5.1.1
```

Comment:
Do you want to just use latest?

Reply:
@blublinsky v5.1.1 is the latest https://github.com/kubernetes-sigs/kustomize/releases?page=1